### ISSUE TITLE
if 'Y' in self.mode

### ISSUE THREAD

**Issue by Note-Liu** [ddlee-cn/MuLUT#9]
if 'Y' in self.mode:
       x = torch.cat([x[:, :, 0, 0], x[:, :, 1, 1], x[:, :, 1, 2], x[:, :, 2, 1]], dim=1)

Why is it fixed to take (0,0), (1,1), (1,2), (2,1) here? If the patch is rotated, is it still the same dimension?
